.github/workflows/build.yaml

```diff
@@ -1,5 +1,10 @@
+---
```

Comment:
I still don't know whether this is recommended or just some convention.... at least it also works without the document separator.

Reply:
> This also serves to signal the start of a document if no [directives](https://yaml.org/spec/1.2.2/#directives) are present.

https://yaml.org/spec/1.2.2/

I understand that it is not only a convention.